src/Careerbuilder.php

```diff
@@ -180,6 +183,72 @@ function ($value, $key) use (&$queryString) {
     }
 
     /**
+     * Get min and max salary numbers from string
+     *
+     * @return array
+     */
+    public function parseSalariesFromString($input = null)
+    {
+        $salary = [
+            'min' => null,
+            'max' => null
+        ];
+        $expressions = [
+            'annualRange' => "/^.\d+k\s-\s.\d+k\/year$/",
+            'annualFixed' => "/^.\d+k\/year$/",
+            'hourlyRange' => "/^.\d+.\d+\s-\s.\d+.\d+\/hour$/",
+            'hourlyFixed' => "/^.\d+.\d+\/hour$/",
+        ];
+
+        foreach ($expressions as $key => $expression) {
+            if (preg_match($expression, $input)) {
+                $method = 'parse'.$key;
+                $salary = $this->$method($salary, $input);
+            }
+        }
+
```

Comment:
It's prettier and [Scrutinizer likes it better(https://scrutinizer-ci.com/g/JobBrander/jobs-careerbuilder/inspections/9166671b-6b93-4177-954a-144ffb74a5be/code-structure/operation/JobBrander%5CJobs%5CClient%5CProviders%5CCareerbuilder%3A%3AparseSalariesFromString). What you think?